### PR TITLE
chore: nightly build

### PR DESCRIPTION
**Pull request by tyliec**
# chore: nightly build

Adds a nightly build to our CI setup.

## Description

TravisCI will run this build nightly and publish any changes. After this is merged in, need to go on TravisCI's web panel and set up the cron job for `develop`.

## Motivation and Context

Continuous testing of our builds.

## Testing

Ran test builds in a test branch.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [X] I have read the **README** document
- [ ] I have added tests to cover my changes
- [X] All new and existing tests passed

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

[issues]: https://github.com/alexa-games/skill-flow-builder/issues

### CODE REVIEW

File: packages/sfb-cli/README.md [hunk 1]
```diff
@@ -193,9 +193,9 @@ export class WaveCommand implements Command {
     private readonly stdOutput: StdOutput) {
   }
   public async run() {
-    this.logger.status('Waving...');
+    this.logger.status('Smiling...');
     if (this.smile) {
-      this.logger.status('Smiling as well...');
+      this.logger.status('Waving as well...');
```
Comment (anfazio) [inline packages/sfb-cli/README.md:198]: Was this change intentional?

Comment (tyliec) [reply packages/sfb-cli/README.md:198]: download

It was not, fixing now.

File: .travis.yml [hunk 2]
```diff
@@ -15,9 +15,17 @@ git:
 stages:
   - name: test
   - name: editor
+  - name: nightly
+    if: type = cron AND branch = master
```
Comment (anfazio) [inline .travis.yml:19]: Nightly builds should be based on the develop branch instead

**Review by anfazio — CHANGES_REQUESTED** [review]

File: packages/sfb-cli/lib/manifestUtils.ts [hunk 3]
```diff
@@ -174,7 +174,14 @@ export class ManifestUtils {
             return;
         }
 
-        if (!semver.satisfies(toolingMetadata.version, versionRange)) {
+        const version = semver.coerce(toolingMetadata.version);
```
Comment (anfazio) [inline packages/sfb-cli/lib/manifestUtils.ts:177]: Let's do this as a separate PR as this change should really be going into `develop`

**Review by anfazio — CHANGES_REQUESTED** [review]

**Review by anfazio — APPROVED** [review]

**Pull request merged**